test/tests/auxkernels/higher_order/high_order_over_sample.i

```diff
@@ -0,0 +1,80 @@
+# MOOSE Test Suite
+#-----------------
+#
+# This test demonstrates the usage of higher order auxilary variables. A standard Gaussian function is is calulated on
+# an eighth order monomail basis on a coarse mesh (4x4). Constant or linear elements would be insufficient to capture
+# the curvature of the solution given the number of elements. Oversampling with four levels of refinemint is used to visualize
```

Comment:
Ah crap...

@friedmud - for background one proposal for tracking requirements was to document tests explicitly spelling out the requirements being tested. This has the added benefit of also fulfilling the standard "tracability matrix" used in older style SQA practices which may be closer to what NQA-1 is looking for. Anyway, If you have thoughts on this, I'd like to hear it. I might also propose using some keyword in the header so we could write a script to extract our current requirements set based on these "standardized" headers.

```
# Requirement: Higher Order Auxiliary Variables
```